src/main/java/rostyslav/ludchenko/Counter.java

```diff
@@ -0,0 +1,18 @@
+package rostyslav.ludchenko;
+
+public class Counter {
+    private static int counter = 0;
+    private final int fin = 100;
```

Comment:
```suggestion
    private final int limit = 100;
```
fin - не дуже інформативна назва

Reply:
Done